keps/sig-auth/2579-psp-replacement/README.md

```diff
@@ -273,6 +267,14 @@ field wasn't added until Kubernetes v1.8, and all containers prior to v1.8 impli
 - `true` (equal in privilege to a v1.7 pod that didn't set the field)
 - `false` (strictly less privileged than other allowed values)
 
+<<[UNRESOLVED]>>
+
+_Blocking for Beta._
+
+How long will old profiles be kept for? What is the removal policy?
```

Comment:
I would expect the implementation to keep old policies available ~forever (and be implemented in a way to make that feasible/maintainable)

Reply:
can we clarify this and drop Unresolved? the intent here is to allow cluster upgrade to be completely decoupled from policy changes and let the admin manage policy updates at their own cadence.